### DISCUSSION TITLE
more images = worse result

### DISCUSSION THREAD

**Discussion by LMBits** [JoePenna/Dreambooth-Stable-Diffusion#69] [Q&A]
I am wondering why that is happening. If I use 100 images with 10100 steps I get much worse results than using 20 images at 2020. How come? Anyone?

**Comment by hashnag** [comment]
I believe more images require more training steps. At some point I read somewhere a suggestion to do 100x training images for your total steps.

So that would be 10,000 total and might explain why you had much better results doing 20 training images at 2020 steps (20 x 100=2000).

**Reply by LMBits** [reply]
Oh no I made a mistake and typed 1100 as in reality it was 10100 I wish there was better documentation specific to this matter. Resolutions, number of images, epochs etc. To understand better how it works internally.

**Comment by LMBits** [comment]
What I found is that training for more steps means more color saturation. Not sure why but doing 30 images for 6060 it makes the generations way more saturated to a point beyond the artist range. Specially if you up CFG scale which normally tends to make things more saturated by design. Not sure why that is.
